src/components/icons/Preloader/Preloader.tsx

```diff
@@ -0,0 +1,431 @@
+import classes from './Preloader.module.scss';
+
+const Preloader = (): JSX.Element => (
+  <div className={classes.preloader}>
+    <svg
```

Comment:
it is toooooo big component)) at first if u retur JSX.Element => it is a component so it sgould not leave in some strange folder as icons, second use svgr plugin for using svg as a React components.

1. npm i vite-plugin-svgr --save-dev
2. add to vite.config.js
```
import svgr from 'vite-plugin-svgr'

export default {
  plugins: [svgr({
      exportAsDefault: true,
    })],
}
```
3. Save your svg as a simple image in assets folder
4. use:  
```
import NameThat UouWish from '../../../assets/{name of the svg}.svg';  
```
to import svg in your component
5. us your svg as a React component:
```
import ViteSVG from '../../../assets/vite.svg';

const Preloader = (): JSX.Element => (
  <div className={classes.preloader}>
    <ViteSVG />
  </div>
);
```
7.

Reply:
redone. thanks for the detailed instructions)